pkg/pipelinerunmetrics/metrics.go

```diff
@@ -63,7 +63,7 @@ var (
 
 const (
 	// ReasonCancelled indicates that a PipelineRun was cancelled.
-	ReasonCancelled = "Cancelled"
+	ReasonCancelled = v1.PipelineRunReasonCancelled
```

Comment:
can we use the constant directly instead of redefining it?

Reply:
I'm a bit confused what you mean here-- I think `v1.PipelineRunReasonCancelled` already exists, and I preserved the alias here for back compatibility. Do you mean use `v1.PipelineRunReasonCancelled` on line 236, or remove this const definition?

Reply:
yes to the question

if it's there for backwards compatibility, please add a note so that new ones are not added there 🙏🏾

Reply:
done!